cmd/server/main.go

```diff
@@ -84,6 +85,7 @@ func main() { //nolint: funlen
 		httpClientTLSHandshakeTimeout = fs.Duration("tls-handshake-timeout", 30*time.Second, "The HTTP client TLS handshake timeout.")
 		httpClientMaxIdleConn         = fs.Int("max-idle-conns", 100, "The HTTP client maximum number of idle connections")
 		retryMax                      = fs.Int("max-retry-count", 3, "The retry maximum for sending requests to the webhook")
+		validateDynamicWebhookURL     = fs.Bool("validate-webhook-url", false, "Enforce strict validation of dynamic webhook url")
```

Comment:
hm, shouldn't we use it for both the dynamic webhook as well as for the "static" webhook?

Reply:
@Knappek  Ok, I added validation to static hooks as well.